bot/data/message_repository.py

```diff
@@ -68,3 +68,24 @@ async def check_message(self, message_id: int) -> bool:
         async with aiosqlite.connect(self.resolved_db_path) as db:
             async with db.execute('SELECT * FROM Messages WHERE id = ?', (message_id,)) as c:
                 return await c.fetchone() is not None
+
+    async def get_user_message_count(self, user_id, guild_id) -> int:
+        async with aiosqlite.connect(self.resolved_db_path) as db:
+            try:
+                c = await db.execute('SELECT count(*) FROM Messages WHERE fk_guildId = ? AND fk_authorId = ?', (guild_id, user_id,))
+                return (await c.fetchone())[0]
+            finally:
+                c.close()
+
+    async def get_user_message_count_range(self, user_id, guild_id, days) -> int:
+        async with aiosqlite.connect(self.resolved_db_path) as db:
+            try:
+                c = await db.execute(f'SELECT count(*) FROM Messages WHERE fk_guildId = ? AND fk_authorId = ? AND strftime("%Y-%m-%d", time) >= date("now","-{days} days")', (guild_id, user_id,))
```

Comment:
break this up into a """ string on multiple lines and use ? for the days param too